documentation/source/ptq_qat.md

```diff
@@ -0,0 +1,233 @@
+
+The calibration process is performed using a set of representative data samples, known as the calibration dataset, to ensure that the quantization process preserves the model's accuracy as much as possible. The most commonly supported calibration methods are percentile, max, and entropy, which are available in most deep learning frameworks. By using these methods, the quantization process can adapt the parameters based on the specific characteristics of the model and the calibration dataset, resulting in a more accurate quantized model.
+
+## Post-training Quantization
+
+Post-training quantization (PTQ) is a quantization method where the quantization process is applied to the trained model after it has completed training. The model's weights and activations are quantized from high precision to low precision, such as from FP32 to INT8. This method is simple and straightforward to implement, but it does not account for the impact of quantization during the training process.
+
+### Hybrid quantization
+__PREREQUISITE__: You will need `pytorch_quantization` installed:
+
+```shell
+pip install pytorch-quantization --extra-index-url https://pypi.ngc.nvidia.com
+```
+With SuperGradients, performing hybrid quantization takes just two lines of code, except of the model definition:
+
+```python
+import super_gradients.training.models
+from super_gradients.training.utils.quantization.selective_quantization_utils import SelectiveQuantizer
+
+model = super_gradients.training.models.get(model_name="resnet50", pretrained_weights="imagenet")
+
+q_util = SelectiveQuantizer(
+    default_quant_modules_calibrator_weights="max",
+    default_quant_modules_calibrator_inputs="histogram",
+    default_per_channel_quant_weights=True,
+    default_learn_amax=False,
+    verbose=True,
+)
+q_util.quantize_module(model)
+```
+
+### Selective quantization
+SuperGradients supports selective and partial quantization: skipping modules from quantization, or replacing them with quantization-friendly counterparts. 
+
+Using the API of `SelectiveQuantizer` it is straightforward, and it offers great flexibility:
+
+You can skip modules by their names, or by their types:
+```python
+from torch import nn
+
+q_util.register_skip_quantization(layer_names={
+    "layer1",
+    "layer2.0.conv1",
+    "conv1"
+})
+
+q_util.register_skip_quantization(layer_names={nn.Linear})
+```
+
+You can replace modules with another type, e.g. replace `Bottleneck` with SuperGradients' `QuantBottleneck`:
+
+```python
+from super_gradients.training.models import Bottleneck
+from super_gradients.modules.quantization import QuantBottleneck
+
+q_util.register_quantization_mapping(layer_names={Bottleneck},
+                                     quantized_target_class=QuantBottleneck,
+                                     input_quant_descriptor=QuantDescriptor(...),
+                                     weights_quant_descriptor=QuantDescriptor(...))
```

Comment:
Please describe in detail the API of `QuantDescriptor`. At the moment you use `...` but nobody knows what should be there at all.

Reply:
In general, I think it will be beneficial to present the basic building blocks, together some description of the possible arguments ('max', 'percentile', 'histogram', etc)